src/web/src/components/Posts/GItHubInfoMobile.tsx

```diff
@@ -0,0 +1,51 @@
+import { createStyles, makeStyles, Theme, ListSubheader } from '@material-ui/core';
+import Repos from './Repos';
+import Issues from './Issues';
+import PullRequests from './PullRequests';
+import Commits from './Commits';
+import Users from './Users';
+import { filterGitHubUrls } from './GitHubInfo';
+
+type Props = {
+  ghUrls: string[];
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      display: 'flex',
+      flexDirection: 'column',
+      marginTop: '0',
+      lineHeight: '1',
+      [theme.breakpoints.up('lg')]: {
+        width: '21rem',
+      },
+      color: theme.palette.text.secondary,
+    },
+    GitHubInfoContainer: {
+      margin: '0 0 0 1rem',
+    },
+  })
+);
+
+const GitHubInfoMobile = ({ ghUrls }: Props) => {
+  const classes = useStyles();
+  const { repos, issues, pullRequests, commits, users } = filterGitHubUrls(ghUrls);
+
+  return (
+    <div>
+      <ListSubheader className={classes.root}>
```

Comment:
Can you make this `<ListSubheader component="div">` and the same to every `ListSubheader` you add? We have a number of wrong uses of `ListSubheader` throughout the tree because we like its default style. By default it renders a `<li>`, `li` is meant for lists, we don't want to end up having nested `<li>`. I have fixed some `<ListSubheader component="div">` in [#2594](https://github.com/Seneca-CDOT/telescope/pull/2594) but to avoid this in the future I'll file an issue to replace all usage of `ListSubheader` with a different component while maintaining the same style.